### PR TITLE
Fix some typo in 2017-01

### PR DESCRIPTION
**Pull request by azu**
Fix some editorial issue.

/cc @rwaldron @littledan

### CODE REVIEW

File: es7/2017-01/jan-25.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ AWB: We'll take care of the review tomorrow
...
+IS: Explains the current status of the TC39 Fast-track projects to JTC1. We have various components in ECMA to standardize in ISO. For ECMA-262 (the main ECMAWScript standard), we now have an agreement with ISO that we will not fast track it anymore; instead, the ECMAScript Suite ECMA-414 will supercede it (with normative references also to the needed Ecma standards), and the other redundant standard ISO IS 16262 (which is out of date) will be withdrawn. The only standards in common will be ECMA-414 (on the way to fast track), which happened after the Dec 7 ECMA GA. As you remember ECMA-414 has undergone some minor revisions for ISO requests. We have also provided an explanatory report to ISO (what has been published also as an TC39 document). We have published the document TC39/2016/050. The ISO number of the suite will be ISO-/EC IS  22275. Currently it is registeredasa DIS, but the voting has not started yet. When we change the suite standard, then it will get a new Edition number. ECMA-402 will also remain an Ecma only standard and its latest version is just referenced by the ECMAScript Suite.
```
Comment (azu) [inline es7/2017-01/jan-25.md]: ECMA-114 -> ECMA-414

File: es7/2017-01/jan-26.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ JL: So are we going to remove more things from the main proposals list?
 
 DH: What I am hearing is an emerging consensus around proposals into some separate file that...
 
-JHD: We already have withdrawn proposals, for things that are abandonded,r jected, etc. WE can add something for things which are potentialy still acive. It's more likely that list will contain stage 0 or 1 proposals than 2 or 3. If that is the status of the simd spec, for example, okay
+JHD: We already have withdrawn proposals, for things that are abandonded, rjected, etc. WE can add something for things which are potentialy still acive. It's more likely that list will contain stage 0 or 1 proposals than 2 or 3. If that is the status of the simd spec, for example, okay
```
Comment (abouthiroppy) [inline es7/2017-01/jan-26.md]:
```diff
- WE can add something for things which are potentialy still acive.
+ We can add something for things which are potentially still active. 
```

You should modify `We`, `potentially`, and `active`.

File: es7/2017-01/jan-26.md [hunk 3]
```diff
@@ -856,7 +856,7 @@ AWB: I will always be sad that there isn't a keyword in front of those things bu
 
 DE: I think we could defer commas. If you do put a comma it's... the real motivation is for the decorator proposal to have a more convientint syntax. Yeah, you could do this but I don't see it as being really important.
 
-AWB: YOu've had public feedback on this, right? I believe the public feeback, my sense is there has been fedback o both sides.
+AWB: YOu've had public feedback on this, right? I believe the public feedback, my sense is there has been feedback o both sides.
```
Comment (abouthiroppy) [inline es7/2017-01/jan-26.md]: `You've` ?

File: es7/2017-01/jan-26.md [hunk 4]
```diff
@@ -1054,7 +1054,7 @@ MS: A lot of the confusion goes away if there is an error.
...
 
-JHD: All of the existing complaint code with the spec will have this.foo = bar in the constructor. If I try to use public fields, if I am subclassing a library using a setter on the protoype chain, that will be confusing. Maybe i'll just move stuff into my constructor, but if I do this.foo = bar brings about the problem again. The metnal model is that I think users will hold is that this is something I am taking out of the constructor and putting in the class body.
+JHD: All of the existing complaint code with the spec will have this.foo = bar in the constructor. If I try to use public fields, if I am subclassing a library using a setter on the prototype chain, that will be confusing. Maybe i'll just move stuff into my constructor, but if I do this.foo = bar brings about the problem again. The metnal model is that I think users will hold is that this is something I am taking out of the constructor and putting in the class body.
```
Comment (abouthiroppy) [inline es7/2017-01/jan-26.md]: Maybe `mental`?

File: es7/2017-01/jan-25.md [hunk 5]
```diff
@@ -796,7 +796,7 @@ BE: Let's talk about this constant rumor that WASM will replace JS. It may happe
 
 MM: I've been doing this now for 8 years, which I know is nothing compared to you. Leaving the door open for something to be done cleanly at such a time in the future that someone has the energy to do it. To me, that is a big deal. The difference between decimal without value types in the old days vs your proposal now: You've done the work to understand how value types could grow in such a way that what you are proposing now will be retroactively rationalized into the value type proposal. We don't know if it'll turn out that way but you've paved the way. The idea of holding the door open for something often works. If you're going to stay on the standards committee you have to play the long game.
 
-BE: Good point. THere is some pressure with game developers with mutable vectors and matrices for operators. That's separate proposal. The committee could do operators, literals and value types. 
+BE: Good point. There is some pressure with game developers with mutable vectors and matrices for operators. That's separate proposal. The committee could do operators, literals and value types.
```
Comment (ljharb) [inline es7/2017-01/jan-25.md]: - That's *a* separate
 - comma after "literals"

File: es7/2017-01/jan-26.md [hunk 6]
```diff
@@ -476,7 +476,7 @@ MF: How about gl0bal?
 
 MPT: globalThisPascal
 
-JHD: Global, $global, __global__, anyway, I'll open a bug and we can come together and make a list of possibilities.  This will take a while, since each time we try aname, we'll have to find a browser willing to try it, if it breaks, we'll have to roll it back, hope that trust hasn't been broken enough to try again. Both Safari and FF are willing to try again but I don't want to just try again every two months.
+JHD: `Global`, `$global`, `__global__`, anyway, I'll open a bug and we can come together and make a list of possibilities.  This will take a while, since each time we try aname, we'll have to find a browser willing to try it, if it breaks, we'll have to roll it back, hope that trust hasn't been broken enough to try again. Both Safari and FF are willing to try again but I don't want to just try again every two months.
```
Comment (ljharb) [inline es7/2017-01/jan-26.md]: - aname → a name

File: es7/2017-01/jan-26.md (hunk 3 above)
Comment (ljharb) [inline es7/2017-01/jan-26.md]: - YOu've → You've
 - o both sides → on both sides

File: es7/2017-01/jan-26.md [hunk 7]
```diff
@@ -856,7 +856,7 @@ AWB: I will always be sad that there isn't a keyword in front of those things bu
 
 DE: I think we could defer commas. If you do put a comma it's... the real motivation is for the decorator proposal to have a more convientint syntax. Yeah, you could do this but I don't see it as being really important.
 
-AWB: YOu've had public feedback on this, right? I believe the public feeback, my sense is there has been fedback o both sides.
+AWB: YOu've had public feedback on this, right? I believe the public feedback, my sense is there has been feedback o both sides.
 
 DE: Most people said please stop the propsal, we can stand the sigil
```
Comment (ljharb) [inline es7/2017-01/jan-26.md]: propsal → proposal

**Comment by azu** [comment]
@abouthiroppy @ljharb Thanks for review!
I've fixed that.

File: es7/2017-01/jan-26.md [hunk 8]
```diff
@@ -856,11 +856,11 @@ AWB: I will always be sad that there isn't a keyword in front of those things bu
 
 DE: I think we could defer commas. If you do put a comma it's... the real motivation is for the decorator proposal to have a more convientint syntax. Yeah, you could do this but I don't see it as being really important.
```
Comment (ljharb) [inline es7/2017-01/jan-26.md]: convientint → convenient

File: es7/2017-01/jan-25.md [hunk 9]
```diff
@@ -198,9 +198,9 @@ BT: Yes, even non-contributors would be bound.
 
 AWB: We can kick them out any time we want, they have no rights.
 
-BT: WE don't want to start eject people for no reason
+BT: We don't want to start eject people for no reason
```
Comment (ljharb) [inline es7/2017-01/jan-25.md]: s/eject/ejecting

File: es7/2017-01/jan-26.md [hunk 10]
```diff
@@ -854,13 +854,13 @@ DE: You would be okay if you could do class x=1 and anew line and y=2.
...
 AWB: I will always be sad that there isn't a keyword in front of those things but I would prefer if you're going to go that route that yes, it not include commas.
 
-DE: I think we could defer commas. If you do put a comma it's... the real motivation is for the decorator proposal to have a more convientint syntax. Yeah, you could do this but I don't see it as being really important.
+DE: I think we could defer commas. If you do put a comma it's... the real motivation is for the decorator proposal to have a more convenient syntax. Yeah, you could do this but I don't see it as being really important.
 
-AWB: YOu've had public feedback on this, right? I believe the public feeback, my sense is there has been fedback o both sides.
+AWB: You've had public feedback on this, right? I believe the public feedback, my sense is there has been feedback on both sides.
 
-DE: Most people said please stop the propsal, we can stand the sigil
+DE: Most people said please stop the proposal, we can stand the sigil
```
Comment (ljharb) [inline es7/2017-01/jan-26.md]: `s/can/can't`

File: es7/2017-01/jan-26.md [hunk 11]
```diff
@@ -1054,7 +1054,7 @@ MS: A lot of the confusion goes away if there is an error.
...
 
-JHD: All of the existing complaint code with the spec will have this.foo = bar in the constructor. If I try to use public fields, if I am subclassing a library using a setter on the protoype chain, that will be confusing. Maybe i'll just move stuff into my constructor, but if I do this.foo = bar brings about the problem again. The metnal model is that I think users will hold is that this is something I am taking out of the constructor and putting in the class body.
+JHD: All of the existing complaint code with the spec will have this.foo = bar in the constructor. If I try to use public fields, if I am subclassing a library using a setter on the prototype chain, that will be confusing. Maybe i'll just move stuff into my constructor, but if I do this.foo = bar brings about the problem again. The mental model is that I think users will hold is that this is something I am taking out of the constructor and putting in the class body.
```
Comment (ljharb) [inline es7/2017-01/jan-26.md]: s/complaint/compliant

**Comment by azu** [comment]
> Sorry for the continued comments :-)

No problem. I've continued to work for [2 years](https://github.com/rwaldron/tc39-notes/commits?author=azu) :)

**Comment by azu** [comment]
@rwaldron Ping

**Comment by leobalter** [comment]
Rick is in a paternity leave, he'll be back in less than 2 weeks. I don't
think this is urgent, we can wait a bit.

**Pull request merged**